docs/tutorials/thermodynamics_utils/partition_function.py

```diff
@@ -301,9 +301,9 @@ def eval_translational(self, p, t):
         """
         m = sum(self.molecule.masses)
 
-        debrog = (const.H_J_S**2 / (2 * np.pi * m * const.KB_J_PER_K * t)) ** (1 / 2)
+        debrog = (const.H_J_S ** 2 / (2 * np.pi * m * const.KB_J_PER_K * t)) ** (1 / 2)
```

Comment:
It looks like you ran `make black` once while having an outdated `black` installed which is causing the CI failures. You should update your installation, run `make black` and hopefully that should resolve the CI errors.

Reply:
Thank you!